gameserver/controllers/game.ctrl.js

```diff
@@ -35,12 +39,16 @@ function GameCtrl() {
       GameManager.init(game);
     }
 
-    game.ready.push(player);
-    Placement.update(game, positions);
+    if (Placement.update(game, positions) === -1) {
+      console.log('placement on inaccessible tile'.red);
+      player.socket.emit('placement failed');
+    } else {
```

Comment:
Mettre un return; au dessus plutôt qu'un else (http://tomasmuller.com.br/2008/10/26/rule-2-dont-use-the-else-keyword/)